cnxarchive/views/sitemap.py

```diff
@@ -49,21 +53,21 @@ def notblocked(page):
 def sitemap(request):
     """Return a sitemap xml file for search engines."""
     xml = Sitemap()
+    from_id = request.matchdict.get('from_id')
     with db_connect() as db_connection:
         with db_connection.cursor() as cursor:
-            # FIXME
-            # magic number limit comes from Google policy - will need to split
-            # to multiple sitemaps before we have more content
             cursor.execute("""\
-                    SELECT
-                        ident_hash(uuid, major_version, minor_version)
-                            AS idver,
-                        REGEXP_REPLACE(TRIM(REGEXP_REPLACE(LOWER(name),
-                            '[^0-9a-z]+', ' ', 'g')), ' +', '-', 'g'),
-                        revised
-                    FROM latest_modules
-                    WHERE portal_type != 'CompositeModule'
-                    ORDER BY revised DESC LIMIT 50000""")
+                SELECT
+                    ident_hash(uuid, major_version, minor_version)
+                        AS idver,
+                    REGEXP_REPLACE(TRIM(REGEXP_REPLACE(LOWER(name),
+                        '[^0-9a-z]+', ' ', 'g')), ' +', '-', 'g'),
+                    revised
+                FROM latest_modules
+                WHERE portal_type NOT IN ('CompositeModule', 'SubCollection')
+                  AND module_ident >= (SELECT COALESCE(%s, 0))
+                ORDER BY module_ident
```

Comment:
I hunted around a bit, and it's not clear that there _is_ utility in this, but I also recall the original intern doing some research and thinking there was.

Reply:
We can have `sitemap-books-{id}.xml`.

I changed the order because I thought perhaps the old sitemaps can be cached longer, but actually that's wrong... since this is the latest_modules, it's going to change.

Reply:
And the reason I changed `revised` to `module_ident` is, so we can do this in the second sitemap xml easily instead of doing something like `LIMIT 50000 OFFSET 50000` which is apparently very slow compared to something like `module_ident >= 5000`.

Reply:
I'm fine with the numbered sitemaps, and using module_ident. Cacheability is not going to be an issue - these aren't high traffic pages - google/other spyders fetch them maybe once a week.
I think it's not a big deal either way. :)

Reply:
Ok great!  I'll change the order to use `DESC` again, but keep using `module_ident`

Reply:
So it sounds like I don't need to change the order?

Reply:
I think the order probably has no affect, so we can leave it.